user/pages/03.commerce2/02.developer-guide/12.recurring/01.subscriptions-overview/docs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Technical overview
+taxonomy:
+    category: docs
+---
+
+This section of the documentation provides an overview of the Subscription entity and related entities, plugins, and objects. Specifically, this section describes:
+* Billing schedules, the billing schedule plugin, Billing periods, and the Prorater plugin
+* The Subscription type plugin and Charges
+* Recurring order types and order item types
+* The PurchasableEntitySubscriptionTrait and the Initial order
+
+## Billing Schedules
+Each subscription has a billing schedule. Billing schedules are configuration entities that are created through the admin UI: `Commerce >> Configuration >> Payment`. Each billing schedule has a type, which is either prepaid or postpaid (charge at the beginning or at the end of the billing period). 
```

Comment:
We are missing an explanation on what a billing schedule is, a configuration entity that stores settings on how/when a product variation will be billed.